include/sdf/Element.hh

```diff
@@ -378,11 +378,23 @@ namespace sdf
 
     /// \brief Set the include filename to the passed in filename.
     /// \param[in] _filename the filename to set the include filename to.
-    public: void SetInclude(const std::string &_filename);
+    public: void SetInclude(const std::string &_filename) SDF_DEPRECATED(11.0);
 
     /// \brief Get the include filename.
     /// \return The include filename.
-    public: std::string GetInclude() const;
+    public: std::string GetInclude() const SDF_DEPRECATED(11.0);
+
+    /// \brief Set the <include> element that was used to load this element.
+    /// This set by the parser on the first element of the included object (eg.
```

Comment:
```suggestion
    /// This is set by the parser on the first element of the included object (eg.
```

Reply:
Fixed in https://github.com/osrf/sdformat/pull/509/commits/51d0c0f175171d33faf032f8eac37532697efd50